stock_mts_mto_rule/__manifest__.py

```diff
@@ -12,5 +12,6 @@
     "application": False,
     "installable": True,
     "depends": ["stock"],
+    "maintainers": ["InsanivimusOmnes", "CetmixGitDrone"],
```

Comment:
@InsanivimusOmnes Do you really want to adopt this module as I cannot see any contribution for this module apart making fragments ?